src/components/map/KmlService.js

```diff
@@ -153,7 +153,7 @@ goog.require('ga_urlutils_service');
           // OL applies a default scale multiplier to 0.5
           // https://github.com/openlayers/ol3/blob/master/src/ol/format/kml.js#L622
           // https://github.com/openlayers/ol3/pull/5745
-          if (image && image.getScale()) {
+          if (image && image.getScale() && image.getScale() != 1) {
```

Comment:
In the last version they don't take the square root and divide by 2 by default. 

so we need to multiply by 2 to have the scale in the KML file and then take the square root like before.